spec/facades/trivia_facade_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe TriviaFacade do
+  describe 'trivia' do
+    it 'returns a trivia question', :vcr do
+      question = TriviaFacade.trivia
+
+      expect(question.first).to be_a TriviaQuestion
```

Comment:
maybe add a few more expectations here just to verify what kind of data it is? like
`expect(question.first).to have_key :answers`
not sure if that's overkill because it should be tested in the poro too, so maybe we don't actually need it

Reply:
Great call! Feels way better having this explicitly tested even if the poro is testing the same.